README.md

```diff
@@ -39,30 +39,118 @@ extern crate rss;
 
 ## Reading
 
-A channel can be read from any object that implements the `BufRead` trait. 
+### From a `BufRead`
+
+A channel can be read from any object that implements the `BufRead` trait.
 
 ```rust
-let reader: BufRead = ...;
+use std::fs::File;
+use std::io::BufReader;
+use rss::Channel;
+
+let file = File::open("tests/data/rss2sample.xml").unwrap();
+let reader = BufReader::new(file);
 let channel = Channel::read_from(reader).unwrap();
 ```
 
+### From a URL
+
+A channel can also be read from a URL.
+
+To enable this functionality you must enable the `from_url` feature in your Cargo.toml.
+
+```toml
+[dependencies]
+rss = { version = "*", features = ["from_url"] }
+```
+
+```ignore
```

Comment:
This should be  ` ```rust,no_run`

we we want the Rust syntax highlighting, but we don't want this test to run since it hits the network

https://doc.rust-lang.org/nightly/book/first-edition/documentation.html#running-documentation-tests